<div class="chatlog__message-group">
  <div id="chatlog__message-container-1173890640594092062" class="chatlog__message-container" data-message-id="1173890640594092062">
    <div class="chatlog__message">
      <div class="chatlog__message-aside"><img class="chatlog__avatar" src="https://cdn.discordapp.com/avatars/291057993686974466/459a7a7d92d588e0a6d0e6b28373eb7c.png?size=512" alt="Avatar" loading="lazy"></div>
      <div class="chatlog__message-primary">
        <div class="chatlog__header"><span class="chatlog__author" style="color:rgb(241,196,15)" title="yutailang0119" data-user-id="291057993686974466">yutailang0119</span> <a href="/channels/375206337937801216?category=main&amp;channel=swift-zatsu&amp;message_id=1173890640594092062"><span class="chatlog__timestamp" title="Tuesday, November 14, 2023 7:42 AM"></span></a><a href="#chatlog__message-container-1173890640594092062">11/14/2023 7:42 AM</a></div>
        <div class="chatlog__content chatlog__markdown"><span class="chatlog__markdown-preserve">Taskの中のtryが自動ではcatchされないのとか？</span></div>
      </div>
    </div>
  </div>
</div>